src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,44 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = Animal.defaultHealth) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static kill(victim) {
+    Animal.alive = Animal.alive.filter(animal => animal !== victim);
+  }
 }
 
+Animal.alive = [];
+Animal.defaultHealth = 100;
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
```

Comment:
If you get rid of the health argument in the Animal constructor, this would be just `super(name)`